lib/tokeniser.js

```diff
@@ -15,6 +15,22 @@ const tokenRe = {
   "other": /[^\t\n\r 0-9A-Za-z]/y
 };
 
+export const typeNameKeywords=[
```

Comment:
```suggestion
export const typeNameKeywords = [
```

Make sure the code format be consistent with other lines 😁